app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -40,19 +39,10 @@ def sync_repos_to_capsule(capsule_content, repository_ids)
           concurrence do
             repository_ids.each do |repo_id|
               sequence do
-                repo = ::Katello::Repository.where(:pulp_id => repo_id).first ||
-                       ::Katello::ContentViewPuppetEnvironment.where(:pulp_id => repo_id).first
-                if repo && repo.content_type != "yum"
-                  # we unassociate units in non-yum repos in order to avoid version conflicts
-                  # during publish. (i.e. two versions of a puppet module in the same repo)
-                  plan_action(Pulp::Consumer::UnassociateUnits,
-                              capsule_id: capsule_content.capsule.id,
-                              repo_pulp_id: repo_id)
-                end
-
```

Comment:
This problem that this unassociate units was solving is still present with docker, file, (and maybe ostree?), and we likely still need it there.  

For puppet we do still need to use this new method.